### PR TITLE
feat: add textClassName prop and unit test

### PR DESCRIPTION
**Pull request by Mike-Diaz**
### Description

This adds the `textClassName` prop inside of `Checkbox` in order for custom classes to be applied to the element. This is necessary for styles to target the class name and be applied directly to the children. This does not change the way the component originally works since the standard class will always be applied.

Linked to PR:  https://github.concur.com/coreui/nui-widgets-lab/pull/124

Added Unit tests

### CODE REVIEW

File: src/Forms/Checkbox.test.js [hunk 1]
```diff
@@ -119,6 +119,16 @@ describe('<Checkbox />', () => {
             ).toContain('wonderful-styles');
         });
 
+        test('should set validationClassName on the popover', () => {
```
Comment (skvale) [inline src/Forms/Checkbox.test.js:122]: > should set childrenClassName on the popover

File: src/Forms/Checkbox.js [hunk 2]
```diff
@@ -119,6 +125,8 @@ Please ensure you are either using a visible \`FormLabel\` or an \`aria-label\`
             );
         }
     },
+    /** Additional classes to apply to children */
+    childrenClassName: PropTypes.string,
```
Comment (jbadan) [inline src/Forms/Checkbox.js:129]: I think we have to be a little more careful with the name here - if the child is not a string, it won't add this class. Do we want that? Do we not? I don't know...

File: src/Forms/Checkbox.test.js (hunk 1 above)
Comment (Mike-Diaz) [reply src/Forms/Checkbox.test.js:122]: Oh that is super weird. I was sure I changed the title of this test, thanks!

File: src/Forms/Checkbox.js (hunk 2 above)
Comment (Mike-Diaz) [reply src/Forms/Checkbox.js:129]: I also thought perhaps this was not the best name. Would you have a suggestion of a more semantic, or explicit name? Perhaps `stringChildrenClassName`?

Comment (jbadan) [reply src/Forms/Checkbox.js:129]: I think `textClassName` would be fine

**Review by jbadan — APPROVED** [review]

**Review by jacobdevera — APPROVED** [review]

**Pull request merged**